src/backend/distributed/transaction/remote_transaction.c

```diff
@@ -244,8 +244,7 @@ StartRemoteTransactionCommit(MultiConnection *connection)
 			 * Failing in this state means that we don't know whether the the
 			 * commit has succeeded.
 			 */
-			ReportConnectionError(connection, WARNING);
-			MarkRemoteTransactionFailed(connection, dontRaiseError);
+			HandleRemoteTransactionConnectionError(connection, dontRaiseError);
```

Comment:
Gah, there are apparently a lot of these? Can you make them consistent with the function declaration, i.e. a boolean whose value represents whether to raise errors (this looks like a boolean whose value represents whether to not raise errors).